.gitignore

```diff
@@ -0,0 +1 @@
+*.*~
```

Comment:
It ignores files that end in a tilde, which are temporary files used by *nix operating systems.
(foobar.java~)